README.md

```diff
@@ -27,7 +27,7 @@ Or in your [CocoaPods](http://cocoapods.org/) Podfile:
 # Latest stable release
 pod 'MapboxDirections', '~> 1.1'
 # Latest prerelease
-pod 'MapboxDirections', :git => 'https://github.com/mapbox/mapbox-directions-swift.git', :tag => 'v1.1.0'
+pod 'MapboxDirections', :git => 'https://github.com/mapbox/mapbox-directions-swift.git', :tag => 'v1.2.0-alpha.1'
```

Comment:
Looks like the SPM prerelease line below didn’t get updated. This find-and-replace only looks for the `.exact(…)` syntax:

https://github.com/mapbox/mapbox-directions-swift/blob/8d421c760d3e9bc7bfb6a3a54409b3ca5c98fbe1/scripts/update-version.sh#L43

But we’re using `from:` with SPM because it can respect tadpole-operator semantics even with prerelease tags. So we’d want to replace the second occurrence of `from:` to have the prerelease tag.